tools/mtd-utils/include/linux/types.h

```diff
@@ -10,7 +12,9 @@ typedef uint64_t __u64;
 typedef __u16 __le16;
 typedef __u32 __le32;
 typedef __u64 __le64;
+#ifndef BSD
```

Comment:
Replacing `BSD` with `__FreeBSD__` eliminates the need to include `sys/param.h` and `sys/types.h` above.